tree/dataframe/src/RDFHelpers.cxx

```diff
@@ -47,7 +47,7 @@ void ROOT::RDF::RunGraphs(std::vector<RResultHandle> handles)
    std::vector<RResultHandle> uniqueLoops(s.begin(), s.end());
 
    // Trigger the unique event loops
-   auto run = [](RResultHandle &h) { h.fLoopManager->Run(); };
+   auto run = [](RResultHandle &h) { if (h.fLoopManager) h.fLoopManager->Run(); };
```

Comment:
I'm not sure this should go in this commit/PR, but it's surely better to check than not

Reply:
Indeed, it should have been modified together with the inclusion of the default constructor for `RResultHandle`, but it's justified that it goes together with this commit: since I'm creating empty `RResultHandle`s for `AsNumpy` actions, this check is necessary to run `RunGraphs` (otherwise it segfaults).